sporkfish/searcher/move_ordering/composite_heuristic.py

```diff
@@ -0,0 +1,46 @@
+from enum import IntEnum
+from typing import List
+
+import chess
+
+from ...board.board import Board
+from .killer_move_heuristic import KillerMoveHeuristic
+from .mvv_lva_heuristic import MvvLvaHeuristic
+
+
+# TODO: To be tuned later
+# I'm personally not very big fan of letting thisbe configurable from config
+# People can input whatever nonsensical values
+# But perhaps it could be useful for testing when we first start out. TBD.
+class CompositeHeuristicScore(IntEnum):
+    KILLER_MOVE = 1
+    MVV_LVA = 2
+
```

Comment:
@yibeili  

You're right, this doesn't do anything at the mo because the conditions are mutually exclusive. I actually added this when we didn't add the capturing condition for killer moves then forgot about it - oops. 

But the idea for future is we might want to keep a relative ordering on moves. For example we might say `MVV_LVA < killer_move + history`. Moreover as @ccjeremylo mentioned - we can allow configuration - #120. Then people can specify which move ordering techniques they want (or don't want - by setting weight to 0). 

On second thought IntEnum isn't great for this, because we only allow. I might move this to a dict with floats. Thoughts? Or kill it off entirely?

Reply:
I have changed to a dict so we dont have extra enum, using float values...